### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by yCobanoglu**

### CODE REVIEW

File: ebl/fragmentarium/web/fragment_search.py [hunk 1]
```diff
@@ -36,35 +39,78 @@ def find_needs_revision(_):
...
+        pages: str,
+        paginationIndex: str,
     ) -> FragmentariumSearchQuery:
         parsed_transliteration = (
             self._transliteration_query_factory.create(transliteration)
             if transliteration
             else self._transliteration_query_factory.create_empty()
         )
-        validated_id, validated_pages = self._validate_pages(bibliographyId, pages)
+        validated_id, validated_pages = FragmentSearch._validate_pages(
```
Comment (fsimonjetz) [inline ebl/fragmentarium/web/fragment_search.py:92]: Is there a reason for calling `_validate_pages` via the class as opposed to `self`? (Should we always do one or the other when working with static methods?)

Comment (yCobanoglu) [reply ebl/fragmentarium/web/fragment_search.py:92]: no doesn't matter as far as i know

**Comment by yCobanoglu** [comment]
Code Climate issue can be disregarded in my opinion

**Review by fsimonjetz — APPROVED** [review]
LGTM

**Review by khoidt — APPROVED** [review]
There is one refactoring request by CodeClimate, please satisfy if before you merge.

**Comment by yCobanoglu** [comment]
> There is one refactoring request by CodeClimate, please satisfy if before you merge.

@khoidt it can't be easily refactor i would ignore it as it is a minor issue.

**Pull request merged**